### PR TITLE
Do not use waffle for activation of mobile support due to it prevents…

### PR DESCRIPTION
**Pull request by morenol**
This remove the usage of waffle flags for enable/disable de mobile support in ora2.

A full description of the issue in:

https://discuss.openedx.org/t/upgrading-tutor-for-koa/3499/2

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-5099](https://openedx.atlassian.net/browse/OSPR-5099) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by morenol** [comment]
Hi @pomegranited  and @regisb. Could you kindly review this?

**Review by regisb — APPROVED** [review]
Thanks for the PR @morenol, I appreciate it :) This works for me.

File: openassessment/xblock/test/test_config_mixin.py
Comment (pomegranited) [inline openassessment/xblock/test/test_config_mixin.py:116]: Instead of removing this test completely, could you instead test the feature flag [as is done here](https://github.com/edx/edx-ora2/blob/c12b4851cfbbaea43aee91ddf59d7fcba3dea8b3/openassessment/xblock/test/test_config_mixin.py#L129-L131)?

Want to verify both that this feature is off by default, and on when the feature flag is present is the settings.

**Review by pomegranited — CHANGES_REQUESTED** [review]
Thanks for raising this @morenol !  Will approve once the test is re-added and passing again

File: openassessment/xblock/test/test_config_mixin.py
Comment (pomegranited) [reply openassessment/xblock/test/test_config_mixin.py:116]: Thanks for re-adding that test! It can be a bit simpler though, since it's not as complex as the other tests:
```python
    @ddt.data(
        (True, True),
        (False, False),
        (None, False),
    )
    @ddt.unpack
    def test_mobile_support_enabled(
            self, settings_input, expected_output
    ):
        """
        The mobile support is expected to be enabled only if:
          1) The settings.FEATURES['ENABLE_ORA_MOBILE_SUPPORT'] value is True.
        """
        my_block = MockBlock()
        settings_feature_key = FEATURE_TOGGLES_BY_FLAG_NAME[MOBILE_SUPPORT]
        with self.settings(FEATURES={settings_feature_key: settings_input}):
            self.assertEqual(expected_output, my_block.is_mobile_support_enabled)
```

**Review by pomegranited — APPROVED** [review]
@morenol  This is working for me too!

:+1: once test is simplified.

- [x] I tested this on my devstack and the iOS app, running https://github.com/edx/edx-app-ios/pull/1401.
   - Tested that without enabling the `ENABLE_ORA_MOBILE_SUPPORT` feature flag: ensured that default, mobile-disabled behaviour is maintained.
   - Enabled the `ENABLE_ORA_MOBILE_SUPPORT` feature flag in Studio and LMS, and [restarted memcached, ran `generate_course_blocks`, and logged out and back into the iOS app](https://github.com/edx/edx-ora2/pull/1445#issuecomment-671630071): ensured that the ORA2 component displays in the iOS app.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A, mobile apps
- [ ] Includes documentation: please add the feature flag to [edX Feature Flags](https://openedx.atlassian.net/wiki/spaces/AC/pages/34734726/edX+Feature+Flags) in Confluence.

**Comment by natabene** [comment]
@morenol Thank you for your contribution. @pomegranited Do you want to take it as a Core Committer?

**Comment by pomegranited** [comment]
@natabene  Yep!  I've flagged it as a core committer review, and approved the change.

I don't think there's anything controversial in here, since it maintains the default behavior, and it's still using one of the acceptable feature flag mechanisms. So I'm merging it now. Thank you for reporting the issue so clearly @regisb , and fixing this issue so quickly @morenol !

CC @nedbat @feanil

**Comment by edx-webhook** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-ora2&entry.867055334=https://github.com/edx/edx-ora2/pull/1531&entry.1484655318=https://github.com/morenol&entry.752974735=2020-11-02+15:23&entry.1917517419=2020-11-05+01:13&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
